### PR TITLE
conda-skeleton: Add tests_require

### PR DESCRIPTION
**Pull request by hargup**

### CODE REVIEW

File: conda_build/pypi.py [hunk 1]
```diff
@@ -86,7 +86,8 @@
   # You can also put a file called run_test.py in the recipe that will be run
   # at test time.
 
-  # requires:
+  requires:
+    - {tests_require}
```
Comment (asmeurer) [inline conda_build/pypi.py]: The way `INDENT` works, it already includes the leading spaces and `-`. I think this should be

```
  {requires_comment}requires:{tests_require}
```

where `requires_comment` is `# ` if there are no test requires and empty if there are some.

Comment (hargup) [inline conda_build/pypi.py]: Thanks, fixed it.

**Comment by asmeurer** [comment]
Is there an example package with test_requires that I can test this on?

**Comment by hargup** [comment]
Yes it is the package `fabric`. I should have included that in the commit message.

**Comment by hargup** [comment]
@asmeurer is it good to go?

**Comment by hargup** [comment]
@asmeurer @ilanschnell

**Comment by asmeurer** [comment]
These need to be run through `spec_from_line` so that they are converted to the proper conda format. Right now, `fabric` gives `fudge<1.0` but it should be `fudge <1.0` with the space.

**Comment by hargup** [comment]
@asmeurer fixed that in latest commit.

**Pull request merged**